### PR TITLE
916 TC-WalkerColina-05

### PR DESCRIPTION
**Pull request by wcolina**
**Test Case:**

image

**Automated Test Case Result:**

image

### CODE REVIEW

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEAssignedList.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+public class PPEAssignedList extends BasePage {
+
+    @FindBy(id = "btnNewExistingPpeAssigned")
+    private WebElement newExistingPpeAssignedButton;
+
+    @FindBy(className = "container")
+    private WebElement ppeContainer;
+
+    public PPEAssignedList() {
+        CommonEvents.isPresent(ppeContainer);
```
Comment (airmiky) [inline src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEAssignedList.java]: usar CommonEvents.isVisible(ppeContainer);

File: src/test/resources/features/ppe/AssignedPPEList.feature [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# @Author: Walker Colina
+# feature to review behaviour of assigned ppe list
+Feature: AssignedPPEList
+
+  Scenario: Una vez seleccionado el menu PPE asigned todos los equipos asignados deben ser mostrados en una lista.
+    Given 'SSI-A' home page is loaded
+    And Click 'PPE' menu on 'Header' page
```
Comment (airmiky) [inline src/test/resources/features/ppe/AssignedPPEList.feature]: corregir a minuscula "click"

File: src/test/resources/features/ppe/AssignedPPEList.feature [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+    And Click 'Assigned PPE' sub menu of 'PPE' menu
```
Comment (airmiky) [inline src/test/resources/features/ppe/AssignedPPEList.feature]: corregir a minuscula "click"

**Review by airmiky — COMMENTED** [review]
Revisar las observaciones por favor.

File: src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEAssignedList.java (hunk 1 above)
Comment (wcolina) [reply src/main/java/org/umssdiplo/automationv01/core/managepage/ppe/PPEAssignedList.java]: Done.

File: src/test/resources/features/ppe/AssignedPPEList.feature (hunk 2 above)
Comment (wcolina) [reply src/test/resources/features/ppe/AssignedPPEList.feature]: Done.

File: src/test/resources/features/ppe/AssignedPPEList.feature (hunk 3 above)
Comment (wcolina) [reply src/test/resources/features/ppe/AssignedPPEList.feature]: Done.

**Comment by wcolina** [comment]
All comments fixed.

**Pull request merged**